platform/components/tekton/tasks/patch_trivy.yml

```diff
@@ -1,8 +1,19 @@
+- op: add
+  path: /spec/steps/0
+  value:
+    name: get-latest-cve-findings
+    image: $(params.TRIVY_IMAGE)
+    script: |
+      TRIVY_TEMP_DIR=$(mktemp -d)
+      trivy --cache-dir $TRIVY_TEMP_DIR image --download-db-only
+      tar -cf ./db.tar.gz -C $TRIVY_TEMP_DIR/db metadata.json trivy.db
+      rm -rf $TRIVY_TEMP_DIR
+      tar xvf ./db.tar.gz -C /home/tekton/.cache/trivy/db
```

Comment:
With `make example-golang-pipeline`, I'm getting the following failure:

```
77.12% 1.38 MiB p/s ETA 5s26.04 MiB / 31.01 MiB [----------------------------------------->_______] 
83.96% 1.85 MiB p/s ETA 2s28.32 MiB / 31.01 MiB [-------------------------------------------->____] 
91.30% 1.85 MiB p/s ETA 1s30.33 MiB / 31.01 MiB [----------------------------------------------->_] 
97.81% 1.85 MiB p/s ETA 0s31.01 MiB / 31.01 MiB [---------------------------------------------------] 
100.00% 1.33 MiB p/s 24star: can't change directory to '/home/tekton/.cache/trivy/db': No such file or directory                                                      
                                                                                                         
[trivy-scan-local-fs : trivy-scan] 2023/01/12 15:49:55 Skipping step because a previous step failed
```

Reply:
Will take a look at it ASAP, strange, I should be creating a directory before untar the file 🤔